v22.2/error-handling-and-troubleshooting.md

```diff
@@ -71,18 +71,50 @@ For more information about how to troubleshoot cluster-level issues, see [Troubl
 
 ### High client CPU load or connection pool exhaustion when SCRAM Password-based Authentication is enabled
 
-When [SASL/SCRAM-SHA-256 Secure Password-based Authentication](security-reference/scram-authentication.html) (SCRAM Authentication) is enabled on a cluster, some additional CPU load is incurred on client applications, which are responsible for handling SCRAM hashing. It's important to plan for this additional CPU load to avoid performance degradation, CPU starvation, and connection pool exhaustion on the client. For example, the following set of circumstances can exhaust the client application's resources:
++ [Overview](#overview)
++ [Mitigation steps while keeping SCRAM enabled](#mitigation-steps-while-keeping-scram-enabled)
++ [Downgrade from SCRAM authentication](#downgrade-from-scram-authentication)
 
-1. SCRAM Authentication is enabled on the cluster.
-1. The client driver's connection pool has no defined maximum number of connections.
-1. The client application issues transactions concurrently.
+#### Overview
+
+When [SASL/SCRAM-SHA-256 Secure Password-based Authentication](security-reference/scram-authentication.html) (SCRAM Authentication) is enabled on a cluster, some additional CPU load is incurred on client applications, which are responsible for handling SCRAM hashing. It's important to plan for this additional CPU load to avoid performance degradation, CPU starvation, and [connection pool](connection-pooling.html) exhaustion on the client. For example, the following set of circumstances can exhaust the client application's resources:
```

Comment:
one thing we saw in a support ticket is that connection latency increased, even though application CPU usage was still fine. perhaps we can mention that as another possible symptom

Reply:
Added some text about this!